cityofaustin/atd-moped
# Add new project tags and new component tags

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/16673

I am going to address the entity updates in a subsequent PR

## Testing

must test local

**Steps to test:**

Restart your hasura cluster, applying the new migrations. 

You can see the new project tags on the summary page, search for 2025 and if there are results, the migrations applied successfully. 
![Screenshot 2024-04-10 at 5 04 27 PM](https://github.com/cityofaustin/atd-moped/assets/12474808/2a14ff43-41bb-4a08-9277-79c79e9adb8d)



Component tags can be seen on the edit component form
![Screenshot 2024-04-10 at 5 02 48 PM](https://github.com/cityofaustin/atd-moped/assets/12474808/1d537b75-cd06-45b4-a954-59d14d45c930)


---
#### Ship list
